.goreleaser.yml

```diff
@@ -115,11 +115,16 @@ snapcrafts:
     grade: stable
     confinement: strict
     publish: true
+    plugs:
+      personal-files:
+        read:
+        - $HOME/.ttn-lw-stack.yml
```

Comment:
Is this a thing? I don't think I ever saw this configuration file mentioned anywhere and I'm unable to find it in the codebase either. @htdvisser ?
I think we always suggest configuring the stack via environment variables.

Reply:
This is just how our config package works. It reads `.<binary>.yml` from the workdir, homedir and (xdg) config dir.

Reply:
Though the documentation suggestions environment variables, the stack does search for config in `.ttn-lw-stack.yml`